### PR TITLE
monitoringdashboard: add support for xyChart dataSets.targetAxis and y2Axis

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

File: docs/releasenotes/release-1.120.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ output fields from GCP APIs are in `status.observedState.*`
 * `MonitoringDashboard`
   * Added `style` fields to text widgets.
   * Added `sectionHeader` widgets.
+  * Added `dataSets.targetAxis` and `y2Axis` fields to `xyChart` widgets.
```
Comment (yuwenma) [inline docs/releasenotes/release-1.120.md:30]: super nit: alphabetical order to mitigate mergeconflict.

**Review by yuwenma — COMMENTED** [review]
/lgtm
/approve

Looks great! (looks like you need to re-generate the golden log)

File: docs/releasenotes/release-1.120.md (hunk 1 above)
Comment (justinsb) [reply docs/releasenotes/release-1.120.md:34]: Agreed.  I've tried to put the new widgets before the new fields in the widgets, but .. I agree

**Comment by justinsb** [comment]
I'm a little worried that this one is failing because of server-side defaulting causing a diff (on the y1Axis, I think), hence #2169

**Comment by justinsb** [comment]
/hold

The http log is suspicious here...

**Comment by justinsb** [comment]
/hold cancel

I reviewed the full http.log; it looks right.  The diff is confusing though!

**Comment by yuwenma** [comment]
> /hold cancel
> 
> I reviewed the full http.log; it looks right. The diff is confusing though!

Do you mean the `timeSeriesTable`? Agree it's confusing. I think that code is actually merged https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/b5a5611408a8949faae46c2a9744dad4814125b8/pkg/test/resourcefixture/testdata/basic/monitoring/v1beta1/monitoringdashboard/monitoringdashboardfull/_http.log#L784-L809

**Pull request merged**